bin/find-spark-home.cmd

```diff
@@ -32,7 +32,7 @@ if not "x%PYSPARK_PYTHON%"=="x" (
 )
 
 rem If there is python installed, trying to use the root dir as SPARK_HOME
-where %PYTHON_RUNNER% > nul 2>$1
+where %PYTHON_RUNNER% > nul 2>&1
```

Comment:
This was a mistake. It should be `&` otherwise creates an empty file called `$1`.